2022-04-06 15:29:13,865 INFO    MainThread:16501 [wandb_setup.py:_flush():75] Loading settings from /home/jmj3047/.config/wandb/settings
2022-04-06 15:29:13,865 INFO    MainThread:16501 [wandb_setup.py:_flush():75] Loading settings from /home/jmj3047/mj_CNN_tutorial/wandb/settings
2022-04-06 15:29:13,865 INFO    MainThread:16501 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-06 15:29:13,865 INFO    MainThread:16501 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-04-06 15:29:13,865 INFO    MainThread:16501 [wandb_init.py:_log_setup():438] Logging user logs to /home/jmj3047/mj_CNN_tutorial/wandb/run-20220406_152913-2tjkzvr7/logs/debug.log
2022-04-06 15:29:13,866 INFO    MainThread:16501 [wandb_init.py:_log_setup():439] Logging internal logs to /home/jmj3047/mj_CNN_tutorial/wandb/run-20220406_152913-2tjkzvr7/logs/debug-internal.log
2022-04-06 15:29:13,866 INFO    MainThread:16501 [wandb_init.py:init():472] calling init triggers
2022-04-06 15:29:13,866 INFO    MainThread:16501 [wandb_init.py:init():475] wandb.init called with sweep_config: {}
config: {'lr': 0.001, 'model_name': 'MLPWithDropout', 'optimizer_name': 'Adam', 'scheduler_name': 'WarmupLR'}
2022-04-06 15:29:13,866 INFO    MainThread:16501 [wandb_init.py:init():491] re-initializing run, found existing run on stack: 3chxpza2
2022-04-06 15:29:13,868 INFO    MainThread:16501 [wandb_run.py:_finish():1684] finishing run minjeejang/fastcampus_fashion_mnist_tutorials/3chxpza2
2022-04-06 15:29:13,868 INFO    MainThread:16501 [jupyter.py:save_history():432] not saving jupyter history
2022-04-06 15:29:13,868 INFO    MainThread:16501 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-04-06 15:29:13,869 INFO    MainThread:16501 [wandb_init.py:_jupyter_teardown():370] cleaning up jupyter logic
2022-04-06 15:29:13,869 INFO    MainThread:16501 [wandb_run.py:_atexit_cleanup():1865] got exitcode: 0
2022-04-06 15:29:13,872 INFO    MainThread:16501 [wandb_run.py:_restore():1837] restore
2022-04-06 15:29:14,561 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 829
  total_bytes: 829
}

2022-04-06 15:29:15,085 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 829
  total_bytes: 829
}

2022-04-06 15:29:15,404 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 829
  total_bytes: 829
}

2022-04-06 15:29:16,365 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 829
  total_bytes: 7592
}

2022-04-06 15:29:16,472 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 829
  total_bytes: 24157
}

2022-04-06 15:29:16,575 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 829
  total_bytes: 24157
}

2022-04-06 15:29:16,677 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 829
  total_bytes: 24157
}

2022-04-06 15:29:16,779 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 829
  total_bytes: 24157
}

2022-04-06 15:29:16,882 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24157
  total_bytes: 24157
}

2022-04-06 15:29:16,985 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24157
  total_bytes: 24157
}

2022-04-06 15:29:17,087 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24157
  total_bytes: 24157
}

2022-04-06 15:29:17,189 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24157
  total_bytes: 24157
}

2022-04-06 15:29:17,292 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24157
  total_bytes: 24157
}

2022-04-06 15:29:17,969 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24157
  total_bytes: 24157
}

2022-04-06 15:29:18,481 INFO    MainThread:16501 [wandb_run.py:_on_finish():1995] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 24157
  total_bytes: 24157
}
local_info {
}

2022-04-06 15:29:20,391 INFO    MainThread:16501 [wandb_run.py:_footer_history_summary_info():3087] rendering history
2022-04-06 15:29:20,391 INFO    MainThread:16501 [wandb_run.py:_footer_history_summary_info():3116] rendering summary
2022-04-06 15:29:20,393 INFO    MainThread:16501 [wandb_run.py:_footer_sync_info():3044] logging synced files
2022-04-06 15:29:20,398 INFO    MainThread:16501 [wandb_init.py:init():525] starting backend
2022-04-06 15:29:20,398 INFO    MainThread:16501 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-04-06 15:29:20,400 INFO    MainThread:16501 [backend.py:ensure_launched():219] starting backend process...
2022-04-06 15:29:20,476 INFO    MainThread:16501 [backend.py:ensure_launched():224] started backend process with pid: 7962
2022-04-06 15:29:20,491 INFO    MainThread:16501 [wandb_init.py:init():534] backend started and connected
2022-04-06 15:29:20,528 INFO    MainThread:16501 [wandb_run.py:_label_probe_notebook():1082] probe notebook
2022-04-06 15:29:20,529 INFO    MainThread:16501 [wandb_run.py:_label_probe_notebook():1092] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-04-06 15:29:20,529 INFO    MainThread:16501 [wandb_init.py:init():598] updated telemetry
2022-04-06 15:29:20,530 INFO    MainThread:16501 [wandb_init.py:init():629] communicating run to backend with 30 second timeout
2022-04-06 15:29:22,183 INFO    MainThread:16501 [wandb_run.py:_on_init():1922] communicating current version
2022-04-06 15:29:22,251 INFO    MainThread:16501 [wandb_run.py:_on_init():1926] got version response 
2022-04-06 15:29:22,251 INFO    MainThread:16501 [wandb_init.py:init():660] starting run threads in backend
2022-04-06 15:29:27,283 INFO    MainThread:16501 [wandb_run.py:_console_start():1896] atexit reg
2022-04-06 15:29:27,284 INFO    MainThread:16501 [wandb_run.py:_redirect():1769] redirect: SettingsConsole.WRAP
2022-04-06 15:29:27,284 INFO    MainThread:16501 [wandb_run.py:_redirect():1806] Wrapping output streams.
2022-04-06 15:29:27,286 INFO    MainThread:16501 [wandb_run.py:_redirect():1830] Redirects installed.
2022-04-06 15:29:27,286 INFO    MainThread:16501 [wandb_init.py:init():685] run started, returning control to user process
